restli-server/src/main/java/com/linkedin/restli/internal/server/util/MIMEParse.java

```diff
@@ -93,8 +106,12 @@ protected static ParseResults parseMimeType(String mimeType)
     {
       String p = parts[i];
       String[] subParts = StringUtils.split(p, '=');
-      if (subParts.length == 2)
+      if (subParts.length == 2) {
```

Comment:
Why are we doing this manual parsing here instead of using the mimetype parser?

Reply:
This MimeParse utility is used to pick the best match based on "q". Not use why we used a custom parser, one difference is that special handling of "*" accept header (lines 118 to 121). This is not handled in jdk MimeType.